src/app/report/edit-display/edit-display.component.ts

```diff
@@ -293,20 +285,9 @@ export class EditDisplayComponent {
     }
   }
 
-  toggleToolTip(toolTip: MatTooltip): void {
-    toolTip.show();
-    setTimeout(() => toolTip.hide(), 2500);
-  }
-
   setNewReport(message: string) {
     this.editor.setNewReport(message);
   }
 
-  getSelectedNode(): Report | Checkpoint | undefined {
-    const node: Report | Checkpoint | undefined = this.selectedNode;
-    if (!node) {
-      this.toastService.showDanger('Selected report was undefined');
-    }
-    return node;
-  }
+  protected readonly Number: NumberConstructor = Number;
```

Comment:
Please move this to the top of the file just like ReportUtil

Reply:
Added the change.